src/frontend/apps/web/src/App.tsx

```diff
@@ -9,12 +9,27 @@ import {
 
 const dataContext = parseDataContext(document);
 
-const routes: Routes = {
-  demo: lazy(
-    () => import("@apuiavignonuniversity/warren-tdbp/src/pages/Overview"),
+const tdbpRoutes: Routes = {
+  instructor: lazy(
+    () => import("@apuiavignonuniversity/warren-tdbp/src/pages/Instructor"),
+  ),
+  student: lazy(
+    () => import("@apuiavignonuniversity/warren-tdbp/src/pages/Student"),
   ),
 };
 
+const getTdbpRoute = (): React.LazyExoticComponent<() => JSX.Element> => {
+  if (dataContext.is_instructor) {
+    return tdbpRoutes.instructor;
+  } else {
+    return tdbpRoutes.student;
+  }
+};
+
+const routes: Routes = {
+  video: getTdbpRoute(),
```

Comment:
Why the `video` key ?

Reply:
The `demo` route was renamed to `video` in warren for development, but it was a mistake as it forces all packages using warren-core to have a `video` route for the dev view. As it now supports the LTI select page, it is out of date.